src/functions/__tests__/fast-login.test.ts

```diff
@@ -344,4 +391,149 @@ describe("Fast Login handler", () => {
       );
     }
   );
+
+  it(`GIVEN a valid LolliPoP request 
```

Comment:
This test works because the validation of the signature is mocked.

Reply:
what do you mean with this comment? as i understood, the nonce metadata is optional as per RFC and the dependency, but mandatory for this use case.

Reply:
You are right, add just a comment inside the test that explain that the request, even if invalid with a wrong signature calculation, pass the signature validation step because is mocked.

Reply:
added in https://github.com/pagopa/io-functions-fast-login/pull/27/commits/24017f00ff4d81f3fd02f3eba39eac89e1b43890